sql-parser/src/main/java/io/crate/sql/tree/Insert.java

```diff
@@ -32,9 +33,10 @@
     private final Table<T> table;
     private final DuplicateKeyContext<T> duplicateKeyContext;
     private final List<String> columns;
+    @Nullable
     private final Query subQuery;
```

Comment:
As far as I understand subquery can be null, right ?

Reply:
see the comment above